packages/prettym/prettym.0.0.1/opam

```diff
@@ -0,0 +1,38 @@
+opam-version: "2.0"
+maintainer:   "Romain Calascibetta <romain.calascibetta@gmail.com>"
+authors:      "Romain Calascibetta <romain.calascibetta@gmail.com>"
+homepage:     "https://github.com/dinosaure/prettym"
+bug-reports:  "https://github.com/dinosaure/prettym/issues"
+dev-repo:     "git+https://github.com/dinosaure/prettym.git"
+doc:          "https://dinosaure.github.io/prettym/"
+license:      "MIT"
+synopsis:     "Prettym"
+description:  """A best effort encoder to respect column limitation"""
```

Comment:
The `synopsis` doesn't seem very useful, would it be possible to move the current `description` to the `synopsis` and either removing the description field or elaborating a little bit.

Reply:
Ok, I will try :+1:.